### DISCUSSION TITLE
Does the assistent and it's instructions do anything yet?

### DISCUSSION THREAD

**Discussion by mifriis** [janhq/jan#1666] [Get Help] [answered]
I tried to create a few different assistants, such as one that should be a "software engineer". 

I had to overwrite the "jan" assistent, and it does reflect on new threads, but doesn't seem to have any impact once i write to it in a new thread.

```json
{
  "avatar": "",
  "id": "jan",
  "object": "assistant",
  "created_at": 1705499954409,
  "name": "Engineer",
  "description": "I am an engineer",
  "model": "*",
  "instructions": "You are a software engineer. You see inputs in the context of software development and process around it",
  "file_ids": []
}
```

![image](https://github.com/janhq/jan/assets/1443594/c1f76f74-2db5-422f-9a2f-d7a2bddd69b5)

**Comment by eschewary** [comment]
Bump, I'm having trouble getting Jan to use instructions. I've tried things like: 
-"start every reply with 'your highness'"
-"misspell every word"
-"use a nonsense language"
-etc. 

I'm using models Mistral 7B Q4 and Phi-2 2.2 7B 8Q, both of which I got through the Jan hub.

The only time I've seen success is with "start every reply with 'yes sir'", and the model only followed the directive for 3 responses before giving up. The instruction was still in the field and was never changed. 

I can't tell why Jan is disregarding the instructions, please try to replicate.

**Comment by franconeilglovasa** [comment]
Bump. Got the same issue. Cant make instructions working.

**Comment by ux-han** [comment, marked as answer]
Solved in the lates Jan updates. Please download at www.jan.ai
